src/test/resources/TestNG.xml

```diff
@@ -6,7 +6,13 @@
     <test name="Example Test" preserve-order="false">
         <classes>
             <class name="LoginTests"/>
-            <class name="BaseTest"/>
+            <class name="Homework17"/>
         </classes>
     </test>
+
+</suite>
+<suite name="TestNG">
+<parameter name="BaseURL" value="http://testprto.io/"/>
+<test name="deletePlaylist">
+</test>
```

Comment:
: The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/229273389-76744161-9e08-4aa2-9d71-661d698c7167.png)

HINT

![image](https://user-images.githubusercontent.com/104761482/229273447-a57a0acd-e9f2-446b-9c6c-bdd0e1566381.png)

Reply:
Corrected. But test still fails. Thank you!
<img width="648" alt="Screenshot 2023-04-03 at 11 16 35 AM" src="https://user-images.githubusercontent.com/123701088/229594268-d4b570bc-2a91-466b-8c9b-706ce9dd8e26.png">

Reply:
![image](https://user-images.githubusercontent.com/104761482/230179163-11d31c9c-97c4-4d68-bbef-e9a3c547ff4a.png)